src/idl/include/idl/tree.h

```diff
@@ -207,7 +207,7 @@ enum idl_extensibility {
    the construct itself may be referenced. e.g. @default_nested annotations
    are only referenced by the annotated module, not by any subconstructs */
 #define IDL_ANNOTATABLE(type) \
-  struct { const struct idl_annotation_appl *annotation; type value; }
+  struct { const struct idl_annotation_appl *annotation; type value; bool implicit; }
```

Comment:
This makes the `implicit` flag apply to all annotations. Doesn't it make more sense to add this flag to the extensibility annotation only?

Reply:
I think this flag could also be useful for other annotations, because other (e.g. `@id`, `@nested`) may also have an implicitly set values. So I've added code for setting this flag to `true` in these cases as well.

Reply:
The annotatable was written so that when there's no annotation, the value must be implicit. It makes for a tri-state, not sure if there's a reason that can't be used? Just curious, not a comment.